### PR TITLE
Activity wizard

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

File: src/components/ResourceCard/AddResource/ResourceActivityType.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ let ResourceActivityType = (props) => {
         <div className="activity-item">
           <div
             className="activity-img"
-            style={{ backgroundImage: `url(${activityType.image})`, }}
+            style={{ backgroundImage: `url(${global.config.resourceUrl}${activityType.image})` }}
```
Comment (simonebelotti75) [inline src/components/ResourceCard/AddResource/ResourceActivityType.js:47]: I think you need to add another case for pexels here

File: src/components/ResourceCard/index.js [hunk 2]
```diff
@@ -48,7 +48,10 @@ const ResourceCard = (props) => {
                   <div
                     className="activity-thumb"
                     style={{
-                      backgroundImage: `url(${resource.thumb_url})`,
+
```
Comment (simonebelotti75) [inline src/components/ResourceCard/index.js:51]: Remove this blank line

File: src/components/ResourceCard/index.js [hunk 3]
```diff
@@ -85,7 +88,8 @@ const ResourceCard = (props) => {
                       </Link>
                       <Link
                         className="dropdown-item"
-                        to={`/project/${match.params.projectId}/playlist/${playlist.id}/activity/create/${resource.id}`}
+
```
Comment (simonebelotti75) [inline src/components/ResourceCard/index.js:91]: Remove this blank line

File: src/routers/AppRouter.js [hunk 4]
```diff
@@ -51,6 +51,7 @@ const AppRouter = () => {
         {/* <PublicRoute exact path="/confirm-email/:confirmationId" component={ConfirmPage} /> */}
 
         <PrivateRoute exact path="/account" component={ProfilePage} />
+        <PrivateRoute exact path="/playlist/preview/:playlistId/resource/:resourceId" component={PreviewPage} previewType="playlist" />
```
Comment (simonebelotti75) [inline src/routers/AppRouter.js:54]: Make it multiline as like other route

Comment (simonebelotti75) [reply src/routers/AppRouter.js:54]: Hmm... why is that moved from bottom to top? Any issue with previous?

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/components/ResourceCard/AddResource/ResourceActivityType.js (hunk 1 above)
Comment (muhammadqamar) [reply src/components/ResourceCard/AddResource/ResourceActivityType.js:47]: these are static images , no need of pexel here

File: src/components/ResourceCard/index.js (hunk 2 above)
Comment (muhammadqamar) [reply src/components/ResourceCard/index.js]: done

File: src/components/ResourceCard/index.js (hunk 3 above)
Comment (muhammadqamar) [reply src/components/ResourceCard/index.js:90]: fixed

File: src/routers/AppRouter.js (hunk 4 above)
Comment (muhammadqamar) [reply src/routers/AppRouter.js]: fixed. these route was commented out before.

Comment (muhammadqamar) [reply src/routers/AppRouter.js]: it as previously in block of commented code. we have removed it from there and paster above

**Pull request merged**